cpp/ql/src/Likely Bugs/Arithmetic/SignedOverflowCheck.qhelp

```diff
@@ -4,26 +4,33 @@
 <qhelp>
 <overview>
 <p>
-Testing for signed integer overflow by adding a
-two signed values together and then comparing the result to one
-of the values is ill-formed since the overflow check is undefined.
-The comparison may produce an unintended result, or may be deleted
-by the compiler entirely.
+When checking for integer overflow, one often writes tests like
```

Comment:
We want to avoid an overly formal tone to documentation.
Change "one often writes" to "you'll often write" or "you may often write".